RELEASING.md

```diff
@@ -0,0 +1,244 @@
+# MusicBrainz Server release process
+
+> Preamble:
+> This document covers steps for releasing new versions of MusicBrainz Server
+> which is performed by maintainers only.
+> It includes discussion about private servers, repositories and tools which
+> other contributors don’t have access to.
+> It is made public for transparency and to allow for improvement suggestions.
+
+## Table of contents
+
+<!-- toc -->
+
+- [Prerequisites](#prerequisites)
+- [Release production](#release-production)
+  * [Update translated messages](#update-translated-messages)
+  * [Merge Git branches](#merge-git-branches)
+  * [Build Docker images](#build-docker-images)
+  * [Announce the deployment](#announce-the-deployment)
+  * [Deploy to production](#deploy-to-production)
+  * [Update websites’ banner](#update-websites-banner)
+  * [Update Jira](#update-jira)
+  * [Blog](#blog)
+  * [Add Git tag](#add-git-tag)
+  * [Release musicbrainz-docker](#release-musicbrainz-docker)
+- [Release beta](#release-beta)
+- [Update test](#update-test)
+
+<!-- tocstop -->
+
+## Prerequisites
+
+1. Perl module Locale::PO
+
+   For updating translations, also part of `production` server features in `cpanfile`.
+
+2. Transifex client
+
+   See the section “[Translations](INSTALL.md#translations)” of the installation documentation.
+
+See the private system administration wiki for additional prerequisites.
+
+## Release production
+
+### Update translated messages
+
+Assuming the source messages were updated when releasing beta (if not, see that under the beta
+process below first!), you need to start by updating the translated messages:
+
+1. Run `./po/update_translations.sh --commit` to download the latest .po files
+   from Transifex and commit them to the `beta` branch.
+
+### Merge Git branches
+
+1. Merge `beta` to `production` (`git merge --log=876423 --no-ff beta`) and push.
+   Wait until [CircleCI](https://circleci.com/gh/metabrainz/musicbrainz-server) and
+   [Jenkins (Selenium)](https://ci.metabrainz.org/job/musicbrainz-server/) are happy
+   with this merge.
+
+2. Merge `production` to `master` and push.
+
+### Build Docker images
+
+Then, you must build new MusicBrainz Server Docker images from Jenkins, go to
+
+1. Go to https://ci.metabrainz.org/job/musicbrainz-docker-images/
+
+2. “Build with Parameters” (in the left), and enter `production` for _IMAGE_BRANCH_.
```

Comment:
We should also build and release a beta image with the updated translations so that they remain in sync, no? :)

Reply:
This is not something we have been doing so far but to backport critical patches from production to beta, mainly because we were lacking bandwidth and having beta translations out-of-sync for one week was not the worse thing about translation at that time.

But yes, it would be better for beta users and make using beta more attractive too. Feel free to do it when you have time.

The translation integration workflow will be drastically changed when switching to Weblate (still at the debug stage for weeks) with more fine-grained automated commits rather than the current batch manual commits.

Reply:
I've always done that when I released for the last few months (build both beta and production and push updates of both), but I guess it makes sense to keep this as-is for now and just consider how the whole thing will work with weblate :)

Reply:
I didn’t know, that’s great :)
Ok to keep it as-is for now.